matrix_client/client.py

```diff
@@ -106,27 +108,6 @@ def global_callback(incoming_event):
     def __init__(self, base_url, token=None, user_id=None,
                  valid_cert_check=True, sync_filter_limit=20,
```

Comment:
Not sure what to make of the undocumented `sync_filter_limit`, which is marked as deprecated in `login` docstring, so I left it undocumented.